djstripe/stripe_objects.py

```diff
@@ -1117,35 +1173,42 @@ def create_token(cls, number, exp_month, exp_year, cvc, **kwargs):
 
 class StripeInvoice(StripeObject):
     """
-Invoices are statements of what a customer owes for a particular billing period, including subscriptions,
-invoice items, and any automatic proration adjustments if necessary.
+    Representation of a Stripe Invoice object.
```

Comment:
"""A Representation of a Stripe <name> object.""" might be the best way to start each class doc. I like this.